.circleci/config.yml

```diff
@@ -31,3 +43,37 @@ workflows:
           filters:
             branches:
               only: master
+
+commands:
+  docker_prepare:
+    description: Downloads the buildx cli plugin and enables multiarch images
+    parameters:
+      buildx_version:
+        type: string
+        default: "v0.4.1"
+    steps:
+      - run: apk add --no-cache curl
+      - run: mkdir -vp ~/.docker/cli-plugins/ ~/dockercache
+      - run: curl --silent -L "https://github.com/docker/buildx/releases/download/<< parameters.buildx_version >>/buildx-<< parameters.buildx_version >>.linux-amd64" > ~/.docker/cli-plugins/docker-buildx
+      - run: chmod a+x ~/.docker/cli-plugins/docker-buildx
+      - run: docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+      - run: docker context create builder
```

Comment:
can you explain why this is required? it appears to create a context that is then unused?

Reply:
... but it's not used in the line below?

Reply:
- run: docker context create builder
      - run: docker buildx create builder --use

We create the context builder, and use the context builder below it, i tried without and it failed, i can do more testing to figure out why that is, appears to be a buildx issue.

Reply:
oh sorry, I've just realised that the argument to `docker buildx create` is the name of the context to use.

I think you could probably get away with 

```
run: docker buildx create --use
```

to use the default context. On the other hand it's fine how it is.

Reply:
im happy to change it, removes a line of code at least. Lets try it

Reply:
ok that looks good.